rpc_jobs/rpc_octavia_gating.yml

```diff
@@ -0,0 +1,46 @@
+ - project:
+    name: "rpc-octavia"
+
+    # This is the URL for the project's repository
+    # which contains the scripts which will be
+    # executed.
+    repo_url: "https://github.com/rcbops/rpc-octavia"
+
+    # This is the whitelist for branch names which
+    # should trigger the job automatically. As this
+    # job only implements PR tests, each item is used
+    # as a Java regex match.
+    branches:
+      - "master"
+
+    # This is a list of names to designate the image
+    # used when preparing a cloud instance. Each name
+    # must also include the key IMAGE with a value
+    # that corresponds to the appropriate image UUID
+    # or name of the image available in the cloud.
+    image:
+      - xenial:
+          IMAGE: "Ubuntu 16.04 LTS (Xenial Xerus) (PVHVM)"
+
+    # This is a list of names to designate the 'scenario'
+    # which is being executed. This provides the ability
+    # to execute the same scripts with different values
+    # which affect the code path tested.
+    # Octavia ignores that
+    scenario:
+      - "functional"
+
+    # This is a list of names to designate the 'action'
+    # which is executed. This provides the ability to
+    # execute the same scripts with different values
+    # for the 'scenario' which affect the code path
+    # tested.
+    # Octavia ignores that
+    action:
+      - "test"
+
+    # This final configuration links this project to the
+    # standard template used.
+    jobs:
+      - 'PR_{name}-{image}-{scenario}-{action}'
```

Comment:
Are you sticking with the gating project PR job?
If you need very different behavior, you may need to define your own job-template and job.
It looks like in the repo you're planning to use this, and the pre/run steps, so I am expecting a yes there.

In that case, shouldn't a ``Execute Pre Script`` stage be included?
Because they aren't defined in the template:
https://github.com/rcbops/rpc-gating/blob/e1a2a69b24e37a93d2a7d2e1b6c0c44f207b3012/rpc_jobs/standard_job_premerge.yml#L33-L46

Reply:
@evrardjp This is implementing the new standard job. Please see https://rpc-openstack.atlassian.net/wiki/spaces/RE/pages/19005457/RE+for+Projects

Reply:
Oh ok. I thought it was ALL steps, not only optional steps.

Reply:
@evrardjp In the template there are no conditional steps, so the STAGES var is now only useful for adjusting the setup/teardown of the single use slave. eg: Adding the ``Pause`` stage.

Reply:
That is a different conversation, shouldn't be on this review.